doc/nrf/config_and_build/sysbuild_migration.rst

```diff
@@ -0,0 +1,579 @@
+| Kconfig option                                                 | Description                                                                 |
++================================================================+=============================================================================+
+| :kconfig:option:`SB_CONFIG_WIFI_NRF700X`                       | Enable Wifi support for nRF700x                                             |
++----------------------------------------------------------------+-----------------------------------------------------------------------------+
+| :kconfig:option:`SB_CONFIG_WIFI_NRF700X_SYSTEM_MODE`           | Use system mode firmware patches and set application to this mode           |
++----------------------------------------------------------------+-----------------------------------------------------------------------------+
+| :kconfig:option:`SB_CONFIG_WIFI_NRF700X_SCAN_ONLY`             | Use scan-only mode firmware patches and set application to this mode        |
++----------------------------------------------------------------+-----------------------------------------------------------------------------+
+| :kconfig:option:`SB_CONFIG_WIFI_NRF700X_RADIO_TEST`            | Use radio test mode firmware patches and set application to this mode       |
++----------------------------------------------------------------+-----------------------------------------------------------------------------+
+| :kconfig:option:`SB_CONFIG_WIFI_NRF700X_SYSTEM_WITH_RAW_MODES` | Use system with raw modes firmware patches and set application to this mode |
++----------------------------------------------------------------+-----------------------------------------------------------------------------+
+| :kconfig:option:`SB_CONFIG_WIFI_PATCHES_EXT_FLASH_DISABLED`    | Load firmware patches directly from ram (default)                           |
++----------------------------------------------------------------+-----------------------------------------------------------------------------+
+| :kconfig:option:`SB_CONFIG_WIFI_PATCHES_EXT_FLASH_XIP`         | Load firmware patches from external flash using XIP                         |
++----------------------------------------------------------------+-----------------------------------------------------------------------------+
+| :kconfig:option:`SB_CONFIG_WIFI_PATCHES_EXT_FLASH_STORE`       | Load firmware patches from external flash into RAM and load to radio        |
++----------------------------------------------------------------+-----------------------------------------------------------------------------+
+
+Applications must be updated to select the required Kconfigs that they need for their applications to work at a sysbuild level, these sysbuild Kconfig options are no longer defaulted or gated depending upon what features that the main application uses so need to be manually set properly. If these are not set than nRF700x functionality will not work.
```

Comment:
```suggestion
You must update your applications to select the required Kconfig options at the Sysbuild level for their applications to work.
These Sysbuild Kconfig options are no longer defaulted or gated depending on the features that the main application uses, so you must set these manually. If these options are not set, nRF700x functionality will not work.

```

Reply:
The first sentence is still not clear "Applications must be updated to select the required Kconfigs that they need for their applications to work at a sysbuild level" what do you mean here?